lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -479,7 +471,8 @@ def pick(x):
             return x
 
     for grant in grants:
-        res = re.match("""GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\5)? ?(.*)""", grant[0])
+        res = re.match(
+            """GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\5)? ?(.*)""", grant[0])
```

Comment:
There seem to be a regression there 🤔 : `\5)` should be `\6)` as far as I remember

Reply:
I confirm: current version in devel reverted this change, probably when I handled conflicts on my own changes.

Reply:
Fix in review https://github.com/ansible/ansible/pull/55292

Reply:
fixed